### PR TITLE
Optional installed apps

### PR DESCRIPTION
**Pull request by fgregg**
this PR

- adjusts the crontab for the chicago scrapers so that the DATABASE_URL environmental variable will reference the database for the staging site for Chicago Councilmatic.

- adjusts pupa_settings to optionally add some django "INSTALLED_APPS" if the DATABASE_URL has 'chicago' in it. This is necessary because the django-councilmatic app needs to be an installed app in order for the creationg of opencivicdata objects to trigger `councilmatic_core1 objects. This won't affect non chicago scrapes.

- add django-councilmatic as a dependency in requirements.txt

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -6,3 +6,4 @@ sh
 pytest==4.2.0
 pytest-mock==1.10.1
 requests-mock==1.5.2
+https://github.com/datamade/django-councilmatic/zipball/master
```
Comment (hancush) [inline requirements.txt]: why not install from the 2.x series on pypi?

Comment (fgregg) [reply requirements.txt]: following the pattern of other dependencies like python-opencivic-data that can affect the models

**Comment by fgregg** [comment]
@hancush I have responded to your comments and requested changes

**Comment by fgregg** [comment]
hi hannah, made a few adjustments here

For the high frequency chicago scrapes, scrape directly into the
production databases. For the nightly scrape, start with a just scraping
the material to the file system then do three different imports into
the production, staging, and OCD API database

I also adjusted the paths of the requirements.txt for github depdendencies. I think the old pattern is deprecated.

I'd appreciate it if you could get to this this morning?

**Pull request merged**